### PR TITLE
fix: write longer bytes block header

### PR DESCRIPTION
**Pull request by leoner**

### CODE REVIEW

File: lib/output.js [hunk 1]
```diff
@@ -268,9 +268,16 @@ proto.defaultWriteObject = function(obj) {
...
 
-
+// Writes block data header.
 proto.writeBlockHeader = function (len) {
-  this.out.put(new Buffer([0x77, len]));
+  if (len <= 0xFF) {
+    // Data blocks shorter than 256 bytes are prefixed with a 2-byte header;
+    this.out.put(new Buffer([0x77, len]));
+  } else {
+    // All others start with a 5-byte header.
```
Comment (fengmk2) [inline lib/output.js]: 贴一下 java 代码或者 java 的官方文档说明对应的链接？方便以后看注释能定位到原因

Comment (fengmk2) [inline lib/output.js]: http://docs.oracle.com/javase/6/docs/platform/serialization/spec/protocol.html#10258 blockdatalong

**Comment by fengmk2** [comment]
+1

**Comment by fengmk2** [comment]
@leoner 我来手动合并

**Comment by fengmk2** [comment]
2.1.2

File: lib/output.js (hunk 1 above)
Comment (leoner) [inline lib/output.js]:
```
 /**
         * Writes block data header.  Data blocks shorter than 256 bytes are
         * prefixed with a 2-byte header; all others start with a 5-byte
         * header.
         */
        private void writeBlockHeader(int len) throws IOException {
            if (len <= 0xFF) {
                hbuf[0] = TC_BLOCKDATA;
                hbuf[1] = (byte) len;
                out.write(hbuf, 0, 2);
            } else {
                hbuf[0] = TC_BLOCKDATALONG;
                Bits.putInt(hbuf, 1, len);
                out.write(hbuf, 0, 5);
            }
        }
```

**Pull request merged**